assets/js/components/Dashboard/PredictionSuppressionPage.tsx

```diff
@@ -24,8 +34,18 @@ const directionNames = (line: string) => {
 };
 
 const PredictionSuppressionPage = () => {
-  const [line, setLine] = useState<string>("Green");
+  const [params, setParams] = useSearchParams();
+  const [line, setLine] = useState<string>(params.get("line") || "Green");
   const { places, lineStops } = useScreenplayContext();
+  const { suppressedPredictions, mutateSuppressedPredictions } =
+    usePredictionSuppressionState();
+
+  useEffect(() => {
+    const newParams = new URLSearchParams();
+    newParams.set("line", line);
+    setParams(newParams);
+  }, [setParams, line]);
```

Comment:
I'm not sure I fully understand what's going on, but I think this is causing some weirdness with how this page handles browser history? But basically, after I select a different line to filter by, going back no longer works. It's almost like the url param is getting overwritten or something it seems.

Reply:
Hmm, strange. It looks like the PA messages page has this problem as well. I suspect it's related to the fact that the setter returned by `useSearchParams` "causes a navigation", according to the docs. I'll experiment with some alternate approaches.